### PR TITLE
[DE] Update `Main Page`

### PR DESCRIPTION
**Pull request by The-Last-Cookie**
Proposal to update the translation of the wiki main page to the latest version.

## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review) (__I'm a native myself__)

### CODE REVIEW

File: wiki/Main_Page/de.md [hunk 1]
```diff
@@ -13,13 +11,19 @@ Willkommen zum osu!-Wiki. Ein Projekt, das eine große Reichweite an osu!-basier
...
+[Regeln](/wiki/Rules) • [Installation](/wiki/Installation) • [Registrierung](/wiki/Registration) • [Hilfe-Center](/wiki/Help_Centre)
+
+[Chat-Konsole](/wiki/Chat_Console) • [BanchoBot](/wiki/BanchoBot) • [FAQ](/wiki/FAQ) • [Tutorials](/wiki/Guides) • [Glossar](/wiki/Glossary)
 
 </div>
 <div class="wiki-main-page-panel">
 
 # Spiel-Client
 
+Als Wegweiser in die Welt des Rhythmus, des Beats und der Blitzlichter hat osu! viel zu präsentieren. Lerne das Spiel in all seinen Einstellungs- und Konfigurationsmöglichkeiten bis hin zu technischen Details, wie z. B. die verschiedenen Dateiformate, besser kennen.
```
Comment (FamousPig) [inline wiki/Main_Page/de.md:25]:
```suggestion
Als Wegweiser in die Welt des Rhythmus, des Beats und der Blitzlichter hat osu! viel zu präsentieren. Lerne das Spiel in all seinen Einstellungs- und Konfigurationsmöglichkeiten bis hin zu technischen Details, wie z. B. den verschiedenen Dateiformaten, besser kennen.
```

File: wiki/Main_Page/de.md [hunk 2]
```diff
@@ -29,42 +33,81 @@ Willkommen zum osu!-Wiki. Ein Projekt, das eine große Reichweite an osu!-basier
...
 
-[Beatmap](/wiki/Beatmap) • [Hit-Objekte](/wiki/Hit_object) • [Mods](/wiki/Game_modifier) • [Punktzahl](/wiki/Score) • [Wiederholung](/wiki/Replay) • [Multi](/wiki/Multi)
+[Beatmap](/wiki/Beatmap) • [Schwierigkeitsgrad](/wiki/Beatmap/Difficulty) • [Hit-Objekte](/wiki/Hit_object) • [Mods](/wiki/Game_modifier) • [Punktzahl](/wiki/Score) • [Wiederholung](/wiki/Replay) • [Medaillen](/wiki/Medals) • [Multi](/wiki/Multi)
 
 </div>
 <div class="wiki-main-page-panel">
 
 # [Beatmap-Editor](/wiki/Beatmap_Editor)
 
+Alle Beatmaps in osu! werden von kreativen Teilnehmern seiner aktiven Community erstellt. Wolltest du schon immer deine Lieblingsmusik mit anderen teilen, aber nie gewusst wie? Mache dich vertraut mit dem eingebauten Editor und die Möglichkeiten, die dieser bietet.
```
Comment (FamousPig) [inline wiki/Main_Page/de.md:47]:
```suggestion
Alle Beatmaps in osu! werden von kreativen Teilnehmern seiner aktiven Community erstellt. Wolltest du schon immer deine Lieblingsmusik mit anderen teilen, hast aber nie gewusst wie? Mache dich vertraut mit dem eingebauten Editor und den Möglichkeiten, die dieser bietet.
```

File: wiki/Main_Page/de.md [hunk 3]
```diff
@@ -29,42 +33,81 @@ Willkommen zum osu!-Wiki. Ein Projekt, das eine große Reichweite an osu!-basier
...
+Was du tun kannst: [Wie und mit was helfen](/wiki/How_you_can_help!) • [Melde schlechtes Verhalten](/wiki/Reporting_Bad_Behaviour) • [Melde Missbrauch](/wiki/Reporting_Bad_Behaviour/Abuse) • [Melde Betrug](/wiki/Reporting_Bad_Behaviour/Handling_Foul_Play)
+
+</div>
+<div class="wiki-main-page-panel">
+
+# Community und Projekte
+
+Eine aktive Community sind eines der Dinge, die osu! voranbringen. Entdecke die Aktivitätan und Veranstaltungen, freunde dich mit anderen an oder helfe, das nächste Event zu organisieren!
+
+[Turniere](/wiki/Tournaments) • [Beatmap Spotlights](/wiki/Beatmap_Spotlights) • [Veranstaltungen](/wiki/Contests) • [Mentorenprogramm](/wiki/Community_Mentorship_Program)
```
Comment (FamousPig) [inline wiki/Main_Page/de.md:100]:
```suggestion
[Turniere](/wiki/Tournaments) • [Beatmap Spotlights](/wiki/Beatmap_Spotlights) • [Wettbewerbe](/wiki/Contests) • [Mentorenprogramm](/wiki/Community_Mentorship_Program)
```

File: wiki/Main_Page/de.md [hunk 4]
```diff
@@ -29,42 +33,81 @@ Willkommen zum osu!-Wiki. Ein Projekt, das eine große Reichweite an osu!-basier
...
+
+[Projekte](/wiki/Projects): [Featured Artists](/wiki/Featured_Artists) • [osu! Community Meetings](/wiki/osu!_Community_Meetings) • [osu!academy](/wiki/osu!academy) • [osu!mapping](/wiki/osu!mapping) • [osu!talk](/wiki/osu!talk)
 
 </div>
 <div class="wiki-main-page-panel">
 
 # Personen
 
+osu! würde nicht ohne die Hilfe von vielen Menschen existieren, die in der Entwicklung, Wartung und des Community-Managements tätig sind. Ihrer Mühen, Zeit und Hingabe ist es zu verdanken, dass das Spiel weiterentwickelt wird.
```
Comment (FamousPig) [inline wiki/Main_Page/de.md:109]:
```suggestion
osu! würde nicht ohne die Hilfe von vielen Menschen existieren, die in der Entwicklung, Wartung und im Community-Management tätig sind. Ihrer Mühen, Zeit und Hingabe ist es zu verdanken, dass das Spiel weiterentwickelt wird.
```

**Review by FamousPig — CHANGES_REQUESTED** [review]

**Review by FamousPig — DISMISSED** [review]

File: wiki/Main_Page/de.md [hunk 5]
```diff
@@ -29,42 +33,81 @@ Willkommen zum osu!-Wiki. Ein Projekt, das eine große Reichweite an osu!-basier
...
+Medienregulierungen: [Richtlinien für die Nutzung von Inhalten](/wiki/Rules/Content_Usage_Guidelines) • [Expliziter Inhalt](/wiki/Rules/Explicit_Content) • [Regeln für Songinhalte](/wiki/Rules/Song_Content_Rules) • [Empfehlungen für visuelle Inhalte](/wiki/Rules/Visual_Content_Considerations)
+
+[Rechtliche Seiten](/wiki/Legal): [Allgemeine Geschäftsbedingungen](/wiki/Legal/Terms) • [Datenschutz](/wiki/Legal/Privacy) • [Urheberrecht (DMCA)](/wiki/Legal/Copyright) • [Musiklizensierung](/wiki/Legal/Music_Licensing)
+
+</div>
+<div class="wiki-main-page-panel">
+
+# Hilfe
+
+
```
Comment (ZeroPyrozen) [inline wiki/Main_Page/de.md:84]: Remove this line

**Review by ZeroPyrozen — APPROVED** [review]

**Review by TicClick — APPROVED** [review]

**Pull request merged**